index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;900&display=swap" rel="stylesheet">
+  <title>Vintage Car Show</title>
+</head>
+<body>
+  <nav class="Navbar">
+    <div class="nav-signin bg-dark d-flex hideOnMobile-f">
+      <ul class="nav-ul gap-close">
+        <li><a href="#" class="fs-medium"><i class="fa fa-facebook fc-white fs-medium"></a></i></li>
+        <li><a href="#" class="fs-medium"><i class="fa fa-twitter fc-white fs-medium"></a></i></li>
+        <li><a href="#" class="fc-white fs-small no-style">Logout</a></li>
+        <li><a href="#" class="fc-white fs-small no-style">Mypage</a></li>
+        <li><a href="#" class="fc-white fs-small no-style">Korean</a></li>
+      </ul>
+    </div>
+    <div class="nav-items d-flex align-center">
+      <button type="button" id="Menu-button">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </button>
+      <div class="nav-brand">
+        <a href="#"><img src="Images/logo.jpg" class="hideOnMobile-f logo"></img></a>
+      </div>
+      <div class="nav-links d-right" id="nav-options">
+        <ul class="nav-ul Mobile-Menu gap-wide display-none" id="mobile-Menu">
+          <li class='d-right'><button type="button" class="hideOnDesktop-I no-style fc-dark" id="close-menu">&times;</button></li>
+          <li><a class="fc-dark fs-medium fc-dark no-style" href="About.html">About</a></li>
+          <li><a class="fc-dark fs-medium fc-dark no-style" href="#Program">Program</a></li>
+          <li><a class="fc-dark fs-medium fc-dark no-style" href="#Guest-Container">Join</a></li>
+          <li><a class="fc-dark fs-medium fc-dark no-style" href="#Sponsor">Sponsor</a></li>
+          <li><a class="fc-dark fs-medium fc-dark no-style" href="#">News</a></li>
+          <li><a class="fc-orange fs-medium padded-border no-style" href="#">CC Campaign</a></li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+
+  <section class="Headline bg-light">
+    <p class="Headline-title">Hello! Car Enthusiasts</p>
+    <h1 class="h1-mobile Headline-title">Miami Vintage Car Show</h1>
```

Comment:
- You should try as much as possible to stick to [the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). For this case I suggest that you insert the image inside the heading text to match the original design ☺️.

Your design 😎 
![image](https://github.com/hamidazim321/Microverse-Capstone-1/assets/99394374/60aebbcc-0ca5-4f6e-b0d5-3ea524e92224)

Original design 😎 
![image](https://github.com/hamidazim321/Microverse-Capstone-1/assets/99394374/a550415b-7594-4623-bbf0-816b1b8efa5b)